src/zserio/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
Consider to not check the runtime in to avoid redundancy/make updating less error-prone.